src/main/java/org/mitre/synthea/export/BB2RIFExporter.java

```diff
@@ -173,35 +177,47 @@ final void prepareOutputFiles() throws IOException {
    * @throws IOException if something goes wrong
    */
   public void export(Person person, long stopTime) throws IOException {
-    exportBeneficiary(person, stopTime);
-    //    exportBeneficiaryHistory(person, stopTime);
-    //    exportOutpatient(person, stopTime);
-    exportInpatient(person, stopTime);
-    //    exportCarrier(person, stopTime);
-    //    exportPrescription(person, stopTime);
+    boolean useConfig = true;
+    exportBeneficiary(person, stopTime, useConfig);
+    exportBeneficiaryHistory(person, stopTime, useConfig);
+    exportInpatient(person, stopTime, useConfig);
+    exportOutpatient(person, stopTime, useConfig );
+    exportCarrier(person, stopTime, useConfig );
+    exportPrescription(person, stopTime, useConfig);
   }
   
   /**
    * Export a beneficiary details for single person.
    * @param person the person to export
    * @param stopTime end time of simulation
+   * @param useConfig flag to use ExportConfig; use false to use original code, true to use ExportConfig and ExportBuilder
    * @throws IOException if something goes wrong
    */
-  private void exportBeneficiary(Person person, long stopTime) throws IOException {
+  private void exportBeneficiary(Person person, long stopTime, boolean useConfig) throws IOException {
```

Comment:
I'm not sure how useful the useConfig parameter will be in the future. Are there cases where we wouldn't want to use the config file?

Reply:
All of these are for regression testing (I run with the config set to false to get the original output, then set it to true to compare the new output using the config TSV file).  I like to keep these for one more PR, until we no longer need these regression tests.  These, by the way, are the bulk of the coverage test errors.